geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RegionManagementController.java

```diff
@@ -38,8 +39,21 @@
       @RequestBody RegionConfig regionConfig) {
     ClusterManagementResult result =
         clusterManagementService.create(regionConfig, "cluster");
-    return new ResponseEntity<>(result,
-        result.isSuccessful() ? HttpStatus.CREATED : HttpStatus.INTERNAL_SERVER_ERROR);
+
+    HttpStatus httpStatus;
+    Status.Result requestStatus = result.getPersistenceStatus().getStatus();
+    if (requestStatus == Status.Result.NO_OP) {
+      httpStatus = HttpStatus.OK;
+    } else if (result.isSuccessful()) {
+      httpStatus = HttpStatus.CREATED;
+    } else {
+      boolean condition = !result.isSuccessfullyAppliedOnMembers()
```

Comment:
Is there a better name for `condition`?

Is this needed `!result.isSuccessfullyAppliedOnMembers()` ? If so can we reconfigure things to make it obvious why it's needed